src/Chapter3.hs

```diff
@@ -971,6 +1086,26 @@ implement the following functions:
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
 
+data Weekday
+    = Sunday
+    | Monday
+    | Tuesday
+    | Wednesday
+    | Thursday
+    | Friday
+    | Saturday
+    deriving (Show, Eq, Bounded, Enum)
+
+isWeekend :: Weekday -> Bool
+isWeekend w = (maxBound == w) || (minBound == w)
```

Comment:
That's a nice implementation! Though, if you decide to change the data type to the "European" system, it will become incorrect, unlike `nextDay`, which will always work :slightly_smiling_face: